### DISCUSSION TITLE
Ability to hook or extend for integrations or auditing

### DISCUSSION THREAD

**Discussion by kirkpabk** [chocolatey/home#237] [Ideas]
Would like to see enhanced hooks or extendable components to support integrations/notifications/observability.

**Comment by TheCakeIsNaOH** [comment]
Chocolatey does support a form of hooks: https://docs.chocolatey.org/en-us/features/hook
If you have specific hooks you would like, there is a repository here: https://github.com/chocolatey-community/chocolatey-hooks

Is that what you are looking for, or are you looking for something else?
